v4/source/system.yaml

```diff
@@ -646,3 +646,90 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  '/svrbusy':
+    post:
+      tags:
+        - system
+      summary: Set the server busy (high load) flag
+      description: |
+        Marks the server as currently having high load which disables non-critical services such as search, statuses and typing notifications.
+        ##### Permissions
```

Comment:
I would add here, and in the other endpoints that the minimum server version is going to be 5.20

Reply:
Minimum server version 5.20 added to all three endpoints via https://github.com/mattermost/mattermost-api-reference/commit/e26aa85745a0df9b21b3b642d3f7efeb1b05de95